nautobot/core/settings_funcs.py

```diff
@@ -27,8 +28,9 @@ def sso_auth_enabled(auth_backends):
 
 @lru_cache(maxsize=5)
 def _sso_auth_enabled(auth_backends):
+    sso_backend_prefix = getattr(settings, "SOCIAL_AUTH_BACKEND_PREFIX", "social_core.backends")
     for backend in auth_backends:
-        if backend.startswith("social_core.backends"):
+        if backend.startswith(sso_backend_prefix):
```

Comment:
I think it'd be preferable to default `SOCIAL_AUTH_BACKEND_PREFIX = "social_core.backends"` in `settings.py` so that it'll be guaranteed to always be set, avoiding the need for `getattr()` here and making it possible for this to just be:

```python
    for backend in auth_backends:
        if backend.startswith(settings.SOCIAL_AUTH_BACKEND_PREFIX):
```